src/core/InputGroup/InputGroup.tsx

```diff
@@ -32,6 +32,11 @@ export type InputGroupProps = {
    * @default false
    */
   disabled?: boolean;
+  /**
+   * Whether the whole input group is required.
+   * @default false
+   */
+  required?: boolean;
```

Comment:
This prop does nothing (other than the visual change) so the user still needs to set `required` manually to the actual inputs inside the group.

But it's the same as how `disabled` works so I didn't add any details in the JSDoc comment.